src/lib.rs

```diff
@@ -11,14 +11,19 @@ pub enum GitHost {
 }
 
 impl GitHost {
-    pub fn source(&self, package: &Package) -> String {
+    pub fn source(&self, package: &Package, no_bin: bool) -> String {
+        // Expecting binary tarballs to be uploaded with a platform identifier.
+        let platform_identifier = match no_bin {
+            true => "",
+            false => "-x86_64",
+        };
```

Comment:
Under what cases would this be empty?

Reply:
My assumption was that a source tarball wouldn't need the architecture in the filename - it can simply be pkgname-pkgver.tar.gz